lib/rules/rule-empty-line-before/__tests__/index.js

```diff
@@ -98,6 +98,32 @@ testRule(rule, {
 
 testRule(rule, {
   ruleName,
+  config: [ "always", { except: ["after-rule"] } ],
+
+  accept: [ {
+    code: "a {} \n b {}",
+  }, {
+    code: "$var: pink;\n\nb {}",
+    description: "scss variable",
+  }, {
+    code: "@media {}\n\na{}",
+    description: "media rule",
+  } ],
```

Comment:
Could you add tests for nested rules either, please? New option affects both non-nested and nested rules.